content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +251,65 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Scheduler configuration migrations
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1beta1 → v1beta2" %}}
+* With the v1beta2 configuration version, you can use a new score extension for the
+  `NodeResourcesFit` plugin.
+  The new extension combines the functionalities of the `NodeResourcesLeastAllocated`,
+  `NodeResourcesMostAllocated` and `RequestedToCapacityRatio` plugins.
+  For example, if you previously used the `NodeResourceLeastAllocated` plugin, you
+  would instead use `NodeResourcesFit` (enabled by default) and add a `pluginConfig`
+  with a `scoreStrategy` that is similar to:
+  ```yaml
+  apiVersion: kubescheduler.config.k8s.io/v1beta2
+  kind: KubeSchedulerConfiguration
+  profiles:
+  - pluginConfig:
+    - args:
+        apiVersion: kubescheduler.config.k8s.io/v1beta2
+        kind: NodeResourcesFitArgs
+        scoringStrategy:
+          resources:
+          - name: cpu
+            weight: 1
+          type: LeastAllocated
+      name: NodeResourcesFit
+  ```
+
+* The scheduler plugin `NodeLabel` is deprecated; instead, use the [`NodeAffinity`](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/) plugin(enabled by default).
+
+* The scheduler plugin `ServiceAffinity` is deprecated; instead, use the `InterPodAffinity` plugin(enabled by default). You should also add a `pluginConfig` that is similar to:
```

Comment:
I don't think users **need** to add any specific configuration. It doesn't need to be part of the migration guide.